agri-store-react/src/Components/Context/LoginPageContext.js

```diff
@@ -0,0 +1,91 @@
+import axios from "axios";
+import React, { createContext, useContext, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import Toast from "../Toast/Toast";
+
+const loginContext = createContext();
+export const useLoginContext = () => useContext(loginContext);
+
+function LoginPageContext({ children }) {
+  const navigate = useNavigate();
+  const [name, setName] = useState();
+  const [email, setEmail] = useState();
+  const [number, setNumber] = useState();
+  const [password, setPassword] = useState();
+  const [loginData, setLoginData] = useState([]);
```

Comment:
Since there are 5 states it will be better to convert all states of `useState `to `useReducer`